inc/class-wpseo-utils.php

```diff
@@ -746,4 +746,32 @@ public static function is_valid_datetime( $datetime ) {
 		return false;
 	}
 
+	/**
```

Comment:
I think this function either needs a more descriptive comment or a different name.

It looks like it cleans up a url to only it's path and the query string. This is not very obvious from the name of the description.